### ISSUE TITLE
Analyzers not being recognized in Dev17 Preview 6

### ISSUE THREAD

**Issue by akhera99** [dotnet/roslyn#57293]
**Version Used**: 
Version 17.0.0 Preview 6.0 [31815.197.d17.0]
**Steps to Reproduce**:
Insert:
```
public class Class
{
    public void Method()
    {
        int x = 5;
        x = 2;
     }
}
```
**Expected Behavior**:
```
public class Class
{
    public void Method()
    {
        int x = 5;
        x = 2; <----- analyzer on this line indicating redundant assignment
     }
}
```
**Actual Behavior**:
No diagnostic is shown.
